### PR TITLE
Allow passing Saver write_version to 'evaluation_once' and 'evaluatio…

### PR DESCRIPTION
**Pull request by yumaokao**
…n_loop'

Thanks @tedhtchang for the commit https://github.com/tensorflow/tensorflow/commit/2a16133061ba3f8fa60c0338cd629f2211f9b17d to add checkpoint file prefix check with default to **SaverDef.V2**.

However there are some [pre-trained slim models](https://github.com/tensorflow/models/tree/master/research/slim#pre-trained-models) which stored in **SaverDef.V1** format. This will broken [eval_image_classifier.py](https://github.com/tensorflow/models/blob/master/research/slim/eval_image_classifier.py) for these models since there is no way to choose in [slim/evaluation.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/slim/python/slim/evaluation.py).

So a parameter **write_version** is added for **evaluation_once** and **evaluation_loop** which allows programs like [eval_image_classifier.py](https://github.com/tensorflow/models/blob/master/research/slim/eval_image_classifier.py) could tell which format the checkpoint uses.

### CODE REVIEW

**Review by sguada — CHANGES_REQUESTED** [review]
Thanks for the fix, but can you add a test for this in evalutation_test.py?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @sguada: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yumaokao** [comment]
Thanks for the suggestion, I'll update the branch with evalutation_test.py.

**Comment by yumaokao** [comment]
In current master branch, there is no prefix check as described in saver.py, so it's OK now to eval pre-trained slim models.
